glusterd2/commands/volumes/grouped-options.go

```diff
@@ -129,6 +129,28 @@ var defaultGroupOptions = map[string]*api.OptionGroup{
 			{Name: "features.shard", OnValue: "on"},
 			{Name: "user.cifs", OnValue: "off"}},
 		"Enable this profile, if the Gluster Volume is used to store virtual machines"},
+	"profile.db": {"profile.db",
+		[]api.VolumeOption{
+			// enable/disable xlators (depends on volgen support PR #1229)
+			{Name: "open-behind", OnValue: "on"},
+			{Name: "write-behind", OnValue: "off"},
+			{Name: "quick-read", OnValue: "off"},
+			{Name: "read-ahead", OnValue: "off"},
+			{Name: "io-cache", OnValue: "off"},
+			{Name: "readdir-ahead", OnValue: "off"},
+			// as per new volgen, this has been named 'md-cache' instead of its
+			// legacy name 'stat-prefetch'
+			{Name: "md-cache", OnValue: "off"},
+			// FIXME: how should this option be named ? fuse.io-threads ?
+			// https://github.com/gluster/glusterd2/issues/376#issuecomment-424970778
+			{Name: "client-io-threads", OnValue: "on"},
```

Comment:
Do not merge this until this is finalised.

Reply:
@aravindavk Thoughts on this ?

Reply:
once template based volgen PR merges, xlator can be enabled/disabled like this. But only if the xlator name exists in the template.

Reply:
In this specific case, io-threads can be loaded on the brick (default) and also on the client (additional and optional). If a user has to turn on io-threads on the client, what will the option look like ?

Reply:
@aravindavk Can you please clarify ?